cmd/jaeger/internal/extension/remotesampling/factory.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package remotesampling
+
+import (
+	"context"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/extension"
+)
+
+// componentType is the name of this extension in configuration.
+var componentType = component.MustNewType("remotesampling")
```

Comment:
elsewhere we used more readable names like `jaeger_storage_exporter`. Can this be `remote_sampling`?

Reply:
Done